### ISSUE TITLE
YouTube videos & Twitter Links. 

### ISSUE THREAD

**Issue by ChristianTucker** [danielireson/unmediumed#1]
It's come to my attention that twitter links aren't scraped and added as part of the markdown response. I understand that constructing an image similar to that of the medium blog isn't something you can do with markdown, but if we could at-least grab the tweet link that would be great. 

https://unmediumed.com/@LitecoinDotCom/the-flappening-is-complete-46d818b36179
https://medium.com/@LitecoinDotCom/the-flappening-is-complete-46d818b36179

Example.

**Comment by danielireson** [comment]
Agreed that it would be good to pull these into the markdown response. Unfortunately it's not as simple as it first seems though as Medium uses a third-party embedding service called [Embedly](https://embed.ly) which pulls in the media via an iframe. I'm sure it's not impossible to extract the information but it's not as easy as the other elements.
